manifests/base/redis.yaml

```diff
@@ -0,0 +1,498 @@
+---
```

Comment:
This is a big and complicated redis install that is supposed to be templated, not "kustomized". Can't we reboot, start from scratch, and try setting up a simple redis install.

We don't need any complicated settings or HA on it

 Let's start with:
 - one redis statefulset
 - Compatible only with a single replicas
 - Support of PVC so that we have persistant storage.
 
 No need for sentinels, master and all that stuff for now.

Reply:
ok